src/test/stacks/stacks-skipfiles-single-authored-js.txt

```diff
@@ -0,0 +1,3 @@
+bar @ ${workspaceFolder}/web/script.js:6:3
```

Comment:
Why is the first frame not skipped?

Reply:
This is also b/c of having to wait for the debouncer. Fixed